src/planner/ompl/Planner.cpp

```diff
@@ -374,75 +376,78 @@ trajectory::InterpolatedPtr planCRRTConnect(
+                                                                    constraint::ProjectablePtr _boundsProjector, 
+                                                                    double _maxDistanceBtwValidityChecks,
+                                                                    double _timeout, size_t _maxEmptySteps,
+                                                                    trajectory::InterpolatedPtr _originalTraj)
 {
 
-// Step 1: Generate the space information of OMPL type
-auto si = getSpaceInformation(
-      _stateSpace, _interpolator, std::move(_dmetric), std::move(_sampler),
-      std::move(_validityConstraint), std::move(_boundsConstraint),
-      std::move(_boundsProjector), _maxDistanceBtwValidityChecks);
+  // Step 1: Generate the space information of OMPL type
+  auto si = getSpaceInformation(
+        _stateSpace, _interpolator, std::move(_dmetric), std::move(_sampler),
+        std::move(_validityConstraint), std::move(_boundsConstraint),
+        std::move(_boundsProjector), _maxDistanceBtwValidityChecks);
 
-// Get the state space
-auto sspace = ompl_static_pointer_cast<GeometricStateSpace>(
-      si->getStateSpace());
+  // Get the state space
+  auto sspace = ompl_static_pointer_cast<GeometricStateSpace>(
+        si->getStateSpace());
 
+  // Step 2: Convert the AIKIDO trajectory to Geometric Path 
+  ::ompl::geometric::PathGeometric *path = new ::ompl::geometric::PathGeometric(si);
 
-// Step 2: Convert the AIKIDO trajectory to Geometric Path 
-::ompl::geometric::PathGeometric *path = new ::ompl::geometric::PathGeometric(si);
+  for (size_t idx = 0; idx < _originalTraj->getNumWaypoints(); ++idx) 
+  {
+        auto ompl_state = sspace->allocState(_originalTraj->getWaypoint(idx));
+        path->append(ompl_state);
+        sspace->freeState(ompl_state);
+  }
 
-for (size_t idx = 0; idx < _originalTraj->getNumWaypoints(); ++idx) 
-{
-      auto ompl_state = sspace->allocState(_originalTraj->getWaypoint(idx));
-      path->append(ompl_state);
-      sspace->freeState(ompl_state);
-}
+  // Step 3: Use the OMPL methods to simplify the path
+  ::ompl::geometric::PathSimplifier simplifier = ::ompl::geometric::PathSimplifier(si);
 
-// Step 3: Use the OMPL methods to simplify the path
-::ompl::geometric::PathSimplifierPtr simplifier;
-simplifier.reset(new ::ompl::geometric::PathSimplifier(si));
+  // Flag for user to know if shorten was successful
+  bool shorten_success = false;
 
-// Set the parameters for termination of simplification process
-::ompl::time::point const time_before = ::ompl::time::now();
-::ompl::time::point time_current;
-::ompl::time::duration const time_limit = ::ompl::time::seconds(_timeout);
-double empty_steps = 0;
+  // Set the parameters for termination of simplification process
+  ::ompl::time::point const time_before = ::ompl::time::now();
+  ::ompl::time::point time_current;
+  ::ompl::time::duration const time_limit = ::ompl::time::seconds(_timeout);
```

Comment:
Nit: I would slightly prefer to use `std::chrono` here, instead of the `ompl::time` wrapper for it.